ComponentKit/Core/Scope/CKDetectComponentScopeCollisions.mm

```diff
@@ -58,5 +119,17 @@ void CKDetectComponentScopeCollisions(const CKComponentLayout &layout)
       }
     }
   }
+  return CKComponentCollision();
+}
+
+void CKDetectComponentScopeCollisions(const CKComponentLayout &layout) {
+#if CK_ASSERTIONS_ENABLED
+  CKComponentCollision collision = CKReturnComponentScopeCollision(layout);
+  if (collision.hasCollision()) {
+    CKCFailAssert(@"Scope collision. Attempting to create duplicate scope for component: %@ with lca: %@\n%@",
```

Comment:
Yes, actually the day we met, I forgot to ask you that, what should the message should say exactly? But I agree that the message should be as clear as possible.